packages/cli/generators/app/templates/src/controllers/ping.controller.ts.ejs

```diff
@@ -1,14 +1,41 @@
 import {Request, RestBindings, get} from '@loopback/rest';
 import {inject} from '@loopback/context';
 
+const PING_RESPONSE = {
```

Comment:
Can we move this to be below the `controller` **and** add a comment above it stating that it's the response object schema. As is, it's very overwhelming to look at.

Reply:
I don't think the decorator can forward reference the constant.